packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -149,8 +151,11 @@ export default function ClassifierContainer({
   }
 
   useEffect(() => {
-    onMount()
-  }, [])
+    console.log({ cachePanoptesData, hydrated, loaded })
+    if (!hydrated) {
+      hydrateStore()
+    }
+  }, [cachePanoptesData, hydrated])
 
   useEffect(function onAuthChange() {
     if (loaded) {
```

Comment:
`workflowID` is a required prop in the classifier so I think it should be checked here. 

Maybe keep track of the last workflow ID that was passed down from the URL. There’s a bug at the moment where the classifier renders with an undefined workflow if you go to `/classify`.